src/platform/FreeBSD.cc

```diff
@@ -152,6 +155,33 @@ CommandLine FreeBSD::ParseArgs(int argc, char *argv[]) const
 	if (binary < 0)
 		throw PosixError("unable to open executable '" + args[0] + "'");
 
+		char *path;
+		int i=1;
+
+		//Try to preopen all the arguments given.
+		
+		//TODO: Make a policy file for each application and prevent the blind opening of all arguments for applications like echo
+
+		if(strcmp(argv[0] , "file")==0){
```

Comment:
Let's leave out the special `file(1)` logic for now: this is the kind of thing that we'll need once we start implementing policy files.